webui.py

```diff
@@ -886,12 +886,18 @@ def initialise(self):
 
         self.app.queue()
 
-    def run(self, share, nobrowser, lan):
+    def get_auth_value(self, user, password):
+        if not user or not password:
+            return None
+        return (user[0], password[0])
```

Comment:
You're passing it args.user, but returning `args.user[0]`? Is that correct?

Reply:
Yes it's right, argparser returns a list even for params taking a single argument. It won't allow the flag without a positional argument so the first index of the list will always be abailable

Reply:
Ah I see. I'm used to getting a single param and not a list. If it's working this way, then it's ok for me